src/main/java/com/CoachPlan/dto/StudentDTO.java

```diff
@@ -1,13 +1,24 @@
 package com.CoachPlan.dto;
 
+
 public class StudentDTO {
 	
 	private String loginID;
+	public String fullName;
+	public String Position;
```

Comment:
See note above... naming conventions indicate that theses should be lowerCamelCase.